<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Global Setting</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QCheckBox" name="cbReplace">
               <property name="text">
                <string>Automatically Replace</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bReplace">
               <property name="text">
                <string>Replace File</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_11">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QCheckBox" name="cbDeleteKMD">
               <property name="text">
                <string>Force Delete KMD File Berfore Building</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_14">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Architecture</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbBitType">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>32Bit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>64Bit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>All</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QRadioButton" name="rbDebug">
               <property name="text">
                <string>Debug</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rbRelease">
               <property name="text">
                <string>Release</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_16">
         <property name="title">
          <string>Path</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Project Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbProjectPath"/>
             </item>
             <item>
              <widget class="QPushButton" name="bProjectPathBrowser">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Target Path</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbTargetPath"/>
             </item>
             <item>
              <widget class="QPushButton" name="bTargetPathBrowser">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_17">
         <property name="title">
          <string>Quick Build</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>G0M</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="bG0MBuild">
               <property name="text">
                <string>Build G0M</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bG0MUMDBuild">
               <property name="text">
                <string>Build G0M_UMD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bG0MKMDBuild">
               <property name="text">
                <string>Build G0M_KMD</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>G3</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="bG3Build">
               <property name="text">
                <string>Build G3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bG3UMDBuild">
               <property name="text">
                <string>Build G3_UMD</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bG3KMDBuild">
               <property name="text">
                <string>Build G3_KMD</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>FantasyPanel</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="bG0MPanelBuild">
               <property name="text">
                <string>Build Panel_G0M</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="bG3PanelBuild">
               <property name="text">
                <string>Build Panel_G3</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Custom Build</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_12">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Build Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbBuildType">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>GPU Driver</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>FantasyPanel</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>All</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_13">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>GPU Model</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbGPUModel">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <item>
                <property name="text">
                 <string>G0M</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>G3</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_15">
            <property name="title">
             <string/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_14">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Driver Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cbDriverType">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <item>
                <property name="text">
                 <string>UMD</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>KMD</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>All</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bManuallyBuild">
            <property name="text">
             <string>Build Custom</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
